nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -1446,6 +1446,809 @@ def _get_initial_states(self, batchsize):
         return input_states
 
 
+class GreedyMultiblankRNNTInfer(_GreedyRNNTInfer):
```

Comment:
Hmm how about subclassing the impl of GreedRNNTinfer instead of the abstract class? Forward can be avoided redoing then and just override _greedy_decode?

Reply:
Well it's just an option, this is fine too

Reply:
Good suggestion. Done.